### ISSUE TITLE
Running tests from command line crash

### ISSUE THREAD

**Issue by DanTup** [Microsoft/vscode#11523]
- VSCode Version: 1.4
- OS Version: Windows 10

We have tests running fine on AppVeyor, just by doing this:

```shell
npm install
npm run vscode:prepublish
node ./node_modules/vscode/bin/test
```

Everything works just great! However, I want to run the same on my local machine (I'm trying to see if I can get coverage reports, so it'll need a lot of faffing around) but I get this:

```text
> node ./node_modules/vscode/bin/test

### VS Code Extension Test Run ###
Current working directory: M:\Coding\Applications\Dart-Code
Running extension tests: M:\Coding\Applications\Dart-Code\.vscode-test\Code M:\Coding\Applications\Dart-Code\out\test --extensionDevelopmentPath=M:\Coding\Applications\Dart-Code --extensionTestsPath=M:\Coding\Applications\Dart-Code\out\test

[9112:4692:0904/171747:VERBOSE1:crash_service_main.cc(68)] Session start. cmdline is [--reporter-url=https://ticinocrashreporter.azurewebsites.net/crash --application-name=VSCode --v=1]
[9112:4692:0904/171747:VERBOSE1:crash_service.cc(142)] window handle is 003A04F8
[9112:4692:0904/171747:VERBOSE1:crash_service.cc(290)] pipe name is \\.\pipe\VSCode Crash Service
dumps at C:\Users\danny\AppData\Local\Temp\VSCode Crashes
[9112:4692:0904/171747:VERBOSE1:crash_service.cc(294)] checkpoint is C:\Users\danny\AppData\Local\Temp\VSCode Crashes\crash_checkpoint.txt
server is https://ticinocrashreporter.azurewebsites.net/crash
maximum 128 reports/day
reporter is electron-crash-service
[9112:4692:0904/171747:VERBOSE1:crash_service_main.cc(84)] Ready to process crash requests
[9112:2868:0904/171747:VERBOSE1:crash_service.cc(323)] client start. pid = 7984
[9112:2868:0904/171747:VERBOSE1:crash_service.cc(323)] client start. pid = 2656
[1096:9060:0904/171747:VERBOSE1:crash_service_main.cc(68)] Session start. cmdline is [--reporter-url=https://ticinocrashreporter.azurewebsites.net/crash --application-name=VSCode --v=1]
[1096:9060:0904/171747:VERBOSE1:crash_service.cc(142)] window handle is 00170492
[1096:9060:0904/171747:VERBOSE1:crash_service.cc(290)] pipe name is \\.\pipe\VSCode Crash Service
dumps at C:\Users\danny\AppData\Local\Temp\VSCode Crashes
[1096:9060:0904/171747:VERBOSE1:crash_service.cc(294)] checkpoint is C:\Users\danny\AppData\Local\Temp\VSCode Crashes\crash_checkpoint.txt
server is https://ticinocrashreporter.azurewebsites.net/crash
maximum 128 reports/day
reporter is electron-crash-service
[1096:9060:0904/171747:ERROR:crash_service.cc(301)] could not start dumper
Error: Illegal state: missing config
Loading development extension at M:/Coding/Applications/Dart-Code
[M:/Coding/Applications/Dart-Code]: Overwriting extension C:/Users/danny/.vscode/extensions/DanTup.dart-code-0.10.0 with M:/Coding/Applications/Dart-Code.
[9112:4896:0904/171748:VERBOSE1:crash_service.cc(331)] client end. pid = 7984
[9112:4896:0904/171748:VERBOSE1:crash_service.cc(331)] client end. pid = 2656
Tests exited with code: 0
PS M:\Coding\Applications\Dart-Code> ^C
PS M:\Coding\Applications\Dart-Code> _service.cc(490)] session ending..
[9112:4692:0904/171749:VERBOSE1:crash_service.cc(495)] clients connected :2
clients terminated :2
dumps serviced :0
dumps reported :0
[9112:4692:0904/171749:VERBOSE1:crash_service_main.cc(89)] Session end. return code is 0
```

The only significant difference between my machine and AppVeyor that I can think of, is that I have Code installed and they don't (they download it). I don't know if that should make any difference though.

I'm not really sure where to start debugging. Is this expected to work? (I presume so, given it works on AppVeyor?)

**Comment by bpasero** [comment]
@DanTup so where does it crash?

**Comment by DanTup** [comment]
@bpasero Sorry, I thought that output was because of a crash. What's happening is the extension host appears, then vanishes, but there's no output of test results. It's not clear if the tests ran at all.

I tried setting `ELECTRON_NO_ATTACH_CONSOLE` (since we needed that to see the test output on AppVeyor) however that just hid most of the output provided above but still didn't get the test results.

I've tried runninig from x86+x64 PoSh and CMD windows though that didn't seem to make any difference.

If the output above is normal, then the issue might just be capturing the test output. We have it working fine on Travis (Linux + Mac) and AppVeyor, so not sure how to troubleshoot it not appearing here on my Windows 10 desktop.
